tests/unit_tests/snapshots/test_libres_facade/test_summary_collector/summary_collector_1.csv

```diff
@@ -1,5 +1,5 @@
-0,1,2,3,4,"BPR:1,3,8",BPR:445,"BPR:5,5,5",BPR:721,FGIP,FGIPH,FGOR,FGORH,FGPR,FGPRH,FGPT,FGPTH,FOIP,FOIPH,FOPR,FOPRH,FOPT,FOPTH,FWCT,FWCTH,FWIP,FWIPH,FWPR,FWPRH,FWPT,FWPTH,WGOR:OP1,WGOR:OP2,WGORH:OP1,WGORH:OP2,WGPR:OP1,WGPR:OP2,WGPRH:OP1,WGPRH:OP2,WOPR:OP1,WOPR:OP2,WOPRH:OP1,WOPRH:OP2,WWCT:OP1,WWCT:OP2,WWCTH:OP1,WWCTH:OP2,WWPR:OP1,WWPR:OP2,WWPRH:OP1,WWPRH:OP2
```

Comment:
Is the behaviour of the summary collector now changed such that we don't get `BPR:445` and `BPR:721`?

Reply:
Yes, because they are not in storage, but in the configuration, hence they came out as empty before.